packages/perseus-core/src/widgets/matcher/matcher-util.test.ts

```diff
@@ -0,0 +1,52 @@
+import getMatcherPublicWidgetOptions from "./matcher-util";
+
+import type {PerseusMatcherWidgetOptions} from "../../data-schema";
+
+describe("getMatcherPublicWidgetOptions", () => {
+    it("should return left and right arrays with sorted items to remove correct order information", () => {
+        // Arrange
+        const options: PerseusMatcherWidgetOptions = {
+            labels: ["**Claims**", "**Evidence**"],
+            padding: true,
+            orderMatters: false,
+            right: [
+                "Medium-sized stars typically exist for roughly 10 billion years",
+                "The current trajectory of the Earth\u2019s tectonic plate movement",
+                "The life cycle of medium-sized stars includes a red giant stage and ends in a whimper as a white dwarf",
+                "Rapid escalation of greenhouse gas emissions",
+                "The current trajectory of the Milky Way galaxy and those in its immediate proximity",
+            ],
+            left: [
+                "Our Sun will run out of fuel and die in around 5 billion years ",
+                "Plate tectonics will rearrange the continents: the Pacific will narrow, bringing Australia closer to the Americas, and the Atlantic will expand to form the largest of the oceans ",
+                "Our Sun will run out of hydrogen, swell into a red giant, gobble up the inner rocky planets, and then collapse and die ",
+                "Average global temperatures will rise ",
+                "In 3 to 4 billion years, our galaxy will begin a slow collision with its closest large neighbor, Andromeda ",
```

Comment:
Should I simplify these to not be so much text? I just copied the data from the matcher test data file.

Reply:
+1. I think smaller data in tests is better. It makes the tests easier to read and change.

Reply:
Agree with Ben.  Unless the quantity of data is needed for the test, let's trim it down. It doesn't even have to be actual real data. I know when we started writing tests for each widget, we pulled actual assessment items so that we'd know what the data looked like.  Today, I don't think we need to do that anymore.

Reply:
Shortened! :)